keyboards/bbrfkr/dynamis/config.h

```diff
@@ -19,9 +19,9 @@
 #include "config_common.h"
 
 /* key matrix */
-#define MATRIX_ROWS 10
+#define MATRIX_ROWS 11
 #define MATRIX_COLS 7
-#define MATRIX_ROW_PINS { B6, B4, D6, D5, D1, C6, B5, D7, D4, D0 }
+#define MATRIX_ROW_PINS { B6, B4, D6, D5, D1, C6, B5, D7, D4, D0, NO_PIN }
```

Comment:
To clarify, the `NO_PIN` is there to facilitate encoder in the matrix, correct? 

If so, then it needs to be reverted.  It's not the proper way to handle this, especially as VIA now supports the encoder map feature. 

https://docs.qmk.fm/#/feature_encoders?id=encoder-map

This allows for proper handling of all keycodes, without special cases, and for it be configured dynamically, by VIA

Reply:
Thank you for your comment. But, I cannot find the document that the via support encoder map... So, I don't understand to construct a json file for supporting encoder map feature...

https://www.caniusevia.com/docs/configuring_qmk

Reply:
https://github.com/qmk/qmk_firmware/pull/17734

Reply:
Thank you for your sharing! I'll try it...

Reply:
Ok. Now, this code support encoder map.

![image](https://user-images.githubusercontent.com/15024038/187093905-22ce316f-6e44-4026-a86a-61bffc113f7a.png)


The json for via is the following;

```
{
    "name": "dynamis",
    "vendorId": "0x6262",
    "productId": "0x0001",
    "lighting": "qmk_rgblight",
    "matrix": {
        "rows": 10,
        "cols": 7
    },
    "layouts": {
        "labels": [
            "ISO Enter",
            "Split Space",
            "Split Left Shift",
            "Split Right Shift",
            "Split Backspace"
        ],
        "keymap": [
            [
                {
                    "x": 15.5
                },
                "5,0\n\n\n4,1",
                "9,0\n\n\n4,1"
            ],
            [
                {
                    "y": 0.25,
                    "x": 2.5
                },
                "0,0",
                "0,1",
                "0,2",
                "0,3",
                "0,4",
                "0,5",
                "0,6",
                "5,6",
                "5,5",
                "5,4",
                "5,3",
                "5,2",
                "5,1",
                {
                    "w": 2
                },
                "5,0\n\n\n4,0"
            ],
            [
                {
                    "x": 2.5,
                    "w": 1.5
                },
                "1,0",
                "1,1",
                "1,2",
                "1,3",
                "1,4",
                "1,5",
                "1,6",
                "6,6",
                "6,5",
                "6,4",
                "6,3",
                "6,2",
                "6,1",
                {
                    "w": 1.5
                },
                "6,0\n\n\n0,0",
                {
                    "x": 4,
                    "w": 1.25,
                    "h": 2,
                    "w2": 1.5,
                    "h2": 1,
                    "x2": -0.25
                },
                "7,0\n\n\n0,1"
            ],
            [
                {
                    "x": 2.5,
                    "w": 1.75
                },
                "2,0",
                "2,1",
                "2,2",
                "2,3",
                "2,4",
                "2,5",
                "2,6",
                "7,6",
                "7,5",
                "7,4",
                "7,3",
                "7,2",
                {
                    "w": 2.25
                },
                "7,0\n\n\n0,0",
                {
                    "x": 3
                },
                "7,1\n\n\n0,1"
            ],
            [
                {
                    "w": 1.25
                },
                "3,0\n\n\n2,1",
                "3,1\n\n\n2,1",
                {
                    "x": 0.25,
                    "w": 2.25
                },
                "3,1\n\n\n2,0",
                "3,2",
                "3,3",
                "3,4",
                "3,5",
                "3,6",
                "8,6",
                "8,5",
                "8,4",
                "8,3",
                "8,2",
                {
                    "w": 2.75
                },
                "8,1\n\n\n3,0",
                {
                    "x": 0.25
                },
                "9,1",
                {
                    "a": 7
                },
                "e0",
                {
                    "x": 0.25,
                    "a": 4,
                    "w": 1.75
                },
                "8,1\n\n\n3,1",
                "8,0\n\n\n3,1"
            ],
            [
                {
                    "x": 2.5,
                    "w": 1.25
                },
                "4,0",
                {
                    "w": 1.25
                },
                "4,1",
                {
                    "w": 1.25
                },
                "4,2",
                {
                    "w": 2.25
                },
                "4,3\n\n\n1,0",
                {
                    "w": 1.25
                },
                "4,4\n\n\n1,0",
                {
                    "w": 2.75
                },
                "4,5\n\n\n1,0",
                {
                    "w": 1.25
                },
                "4,6",
                {
                    "w": 1.25
                },
                "9,6",
                {
                    "w": 1.25
                },
                "9,5",
                {
                    "x": 0.5
                },
                "9,4",
                "9,3",
                "9,2"
            ],
            [
                {
                    "y": 0.25,
                    "x": 6.25,
                    "w": 6.25
                },
                "4,3\n\n\n1,1"
            ]
        ]
    }
}
```